app/build.gradle

```diff
@@ -18,8 +18,8 @@ android {
 
         minSdkVersion 24
         targetSdkVersion 26
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/e69d75178c774aa47ab87c88028da61f8278c92a/app/build.gradle#L20"></span>Not targeting the latest versions of Android; compatibility <br />
modes apply. Consider testing and updating this version. <br />
Consult the android.os.Build.VERSION_CODES javadoc for details.</td>
    </tr>
  </tbody>
</table>